src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykTreeCellRenderer.kt

```diff
@@ -111,7 +111,14 @@ class SnykTreeCellRenderer : ColoredTreeCellRenderer() {
                 val issue = value.userObject as ScanIssue
                 nodeIcon = SnykIcons.getSeverityIcon(issue.getSeverityAsEnum())
                 val range = issue.range
-                text = "${if (issue.additionalData.hasAIFix) "⚡️" else ""} ${
+                var showIgnoredLabel = false
+                var showAIFix = false
+                if (pluginSettings().isGlobalIgnoresFeatureEnabled && issue.isIgnored == true) {
```

Comment:
The `== true` is redundant if issue.isIgnored is a boolean?

Reply:
`issue.IsIgnored` is an optional boolean, see here: https://github.com/snyk/snyk-intellij-plugin/blob/7a7810841bf5ff1842959f08cd65640c58917cbb/src/main/kotlin/snyk/common/lsp/Types.kt#L31

So basically this check is needed so that if `issue.isIgnored` is null then we treat that as a false. I'm not too good at Kotlin yet so IntelliJ asked me to write the code this way 😂